repos/spack_repo/builtin/packages/libmesh/package.py

```diff
@@ -325,6 +327,11 @@ def configure_args(self):
         else:
             options.append("--enable-tbb=no")
 
+        if "+poly2tri" in self.spec:
+            options.append("--enable-poly2tri=yes")
+        else:
+            options.append("--enable-poly2tri=no")
```

Comment:
@capitalaslash, is this regular `autotools` `configure`? If so, it would normally work with:
```suggestion
        options.append(self.enable_or_disable("poly2tri"))
```
That is, if it supports `--disable-poly2tri` as well as `--enable-poly2tri=no`.

Of course, please test both if you apply this suggestion.
Otherwise, you can replace it with a if expression, I think.